dev/tasks/conda-recipes/.ci_support/linux_ppc64le_cuda_compiler_version11.2.yaml

```diff
@@ -35,18 +35,16 @@ google_cloud_cpp:
 libabseil:
 - '20230125'
 libgrpc:
-- '1.54'
 - '1.56'
 libprotobuf:
-- '3.21'
 - 4.23.3
 lz4_c:
 - 1.9.3
 numpy:
-- '1.21'
+- '1.22'
 - '1.23'
-- '1.21'
-- '1.21'
+- '1.22'
+- '1.22'
```

Comment:
From my understanding this this is generated with smithy but why does numpy contains a list of versions with: `1.22, 1.23, 1.22, 1.22`, when is each one of these versions selected? I see lots of the other dependencies have a single version specified, are those the different versions selected for Python 3.8, 3.9, 3.10 and 3.11?
This is only so I understand what it does :)

Reply:
> I see lots of the other dependencies have a single version specified, are those the different versions selected for Python 3.8, 3.9, 3.10 and 3.11?

Yes, exactly. New python versions generally have a lowest-supported numpy version that's higher than the oldest-still-supported numpy at the time of the release of said python version, so there needs to be some mapping between python and numpy versions.

Reply:
Thanks!